library/linear.lib

```diff
@@ -2514,49 +2606,6 @@ X ~ 8 100 -300 250 U 50 50 1 1 I
 ENDDRAW
 ENDDEF
 #
-# OP179GRT
-#
-DEF OP179GRT U 0 20 Y Y 1 F N
-F0 "U" 0 200 50 H V L CNN
-F1 "OP179GRT" 0 -200 50 H V L CNN
-F2 "TO_SOT_Packages_SMD:SOT-23-5" 0 0 50 H I C CNN
-F3 "" 0 200 50 H I C CNN
-$FPLIST
- SOT*23*
-$ENDFPLIST
-DRAW
-P 4 0 1 10 -200 200 200 0 -200 -200 -200 200 f
-X V+ 2 -100 300 150 D 50 50 0 1 W
-X V- 5 -100 -300 150 U 50 50 0 1 W
-X ~ 1 300 0 100 L 50 50 1 1 O
-X + 3 -300 100 100 R 50 50 1 1 I
-X - 4 -300 -100 100 R 50 50 1 1 I
-ENDDRAW
-ENDDEF
-#
-# OP179GS
```

Comment:
OP179GS deleted.
OP179GS now is an alias of Amp.S.SOIC-8.Nc-Ii-Ni-Pn-Nc-O-Pp-Nc